components/feature/sitepermissions/src/main/res/values/strings.xml

```diff
@@ -36,4 +36,6 @@
     <string name="mozac_feature_sitepermissions_never_allow">Never</string>
     <!-- Title of a dialog to require to save data in persistent storage. %1$s will be replaced with the URL of the current page -->
     <string name="mozac_feature_sitepermissions_persistent_storage_title">Allow %1$s to store data in persistent storage?</string>
-</resources>
\ No newline at end of file
+    <!-- Title of a dialog to use EME. %1$s will be replaced with the URL of the current page -->
+    <string name="mozac_feature_sitepermissions_media_key_system_access_title">Allow %1$s to use EME?</string>
```

Comment:
nit: Lets update to reflect the change discussed on matrix  `Allow to %1$s play DRM-controlled content?` also update the comment for translators :)

Reply:
This was already addressed